crates/accelerate/src/circuit_library/pauli_evolution.rs

```diff
@@ -27,6 +26,8 @@ type Instruction = (
     Vec<Clbit>,
 );
 
+// custom types for a more readable code
+
```

Comment:
Maybe this line was supposed to be removed or back on line 21?

Reply:
done in 076f467